web/src/components/PageSkeleton.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { PageSkeleton } from "@/components/PageSkeleton";
+import { useMockUserData } from "@/test/mock/mockUseUserData";
+import { render, screen } from "@testing-library/react";
+
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+
+describe("<RightSidebarPageLayout />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("does not render report an issues bar if onboarding is not completed", () => {
+    useMockUserData({ formProgress: "UNSTARTED" });
+
+    render(
+      <PageSkeleton>
+        <></>
+      </PageSkeleton>
+    );
+
+    expect(screen.queryByTestId("bugBar")).not.toBeInTheDocument();
+  });
+
+  it("renders report an issues bar after onboarding is completed", () => {
+    useMockUserData({});
```

Comment:
I didn't realize useMockUserData wasn't correctly randomizing values, so ideally this shouldn't work. But as a rule, if we're relying on some property in a test, we should be explicit about setting that property and not relying on what happens to come from the generator.

This should explicitly set `formProgress: "COMPLETED"` if that's what the test is relying on